onnx/backend/test/case/node/__init__.py

```diff
@@ -301,6 +304,22 @@ def merge(
             outputs=outputs_vi,
         )
         kwargs["producer_name"] = "backend-test"
+
+        # replace opset versions with what are specified in function proto
+        for opset_import in func_opset_import:
+            if "opset_imports" in kwargs:
```

Comment:
Wouldn't it be simpler to have the if-condition outside the for-loop and change the else-statement to update `kwargs["opset_imports"] = func_opset_import` in 1 step? It looks a bit confusing, even though I guess it works.

Reply:
The purpose is to append if "opset_imports" does not exist, otherwise update. So it cannot be outside of the loop.

Reply:
took a second look, you are right. I made the correction. Thanks!